### PR TITLE
Navigation and content revisions to podman.io to make them more frien…

### PR DESCRIPTION
**Pull request by jberkus**
…dly to first-time visitors.  This includes: (1) Changing the left-panel navigation to put "get started" and "join commuity" at the top (2) replace "tutorials" with "get started" which includes both installation and initial using podman instructions.  (3) replace "mailing list" with "join community" which includes slack and IRC.  (4) add para about getting started to home page.

Importantly, this means that the installation docs and getting started tutorial will move OUT of the libpod repository and be maintained in this one.  This is an improvement because dumping users into an unexpected github repo when they were looking for instructions is confusing.  It also means that yes, the plan is to move all of the non-hacking documentation to this repo instead of having it in github.com/libpod; look for more PRs from me to do just that.

Please review: @TomSweeneyRedHat

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM
@TomSweeneyRedHat PTAL

File: _layouts/default.html [hunk 1]
```diff
@@ -18,16 +18,15 @@
...
       <header>
-        <h1 class="header"><a href="https://podman.io">Podman</a></h1>
+        <h1 class="header"><a href="{{ site.baseurl }}">Podman</a></h1>
         <p class="header">Manage pods, containers, and container images.</p>
 
         <ul>
-          <li><a class="buttons" href="{{ site.baseurl }}/blogs">Blogs</a>
+          <li><a class="buttons" href="{{ site.baseurl }}/getting-started">Get Started</a>
+          <li><a class="buttons" href="{{ site.baseurl }}/community">Join Community</a>
+          <li><a class="buttons" href="{{ site.baseurl }}/blogs">Blog</a>
```
Comment (TomSweeneyRedHat) [inline _layouts/default.html:27]: Did you mean to make "Blogs" singular here?  I think it should still be plural....

File: _site/asciinema/podman/images/index.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<html>
```
Comment (TomSweeneyRedHat) [inline _site/asciinema/podman/images/index.html:1]: This change will require a change to Podmans commands page (https://github.com/containers/libpod/blob/master/commands.md) which has links to these files in the old location.

File: _site/2018/09/25/new.html [hunk 3]
```diff
@@ -0,0 +1,86 @@
+<!doctype html>
```
Comment (TomSweeneyRedHat) [inline _site/2018/09/25/new.html:1]: I'm not sure why this file moved here instead of _site/blogs/....

**Review by TomSweeneyRedHat — COMMENTED** [review]
@jberkus thanks for the heavy lifting on this one.  I'd a structural change that's not clear to my noggin.  I see all the *.html files got recreated under the new _site directory.  Will there be any adjustments necessary for the _posts directory?  Will that be the location where we'll drop *.md files going forward and will they then generate the html but now somewhere under the _site directory?

If someone linked one of our blogs a few weeks ago, will it be a dead link when this PR goes in?  Or is that why you recreated rather than moved the html's?

I'd like to hear from @baude and @mheon about moving the install guide and tutorial here from the libpod GitHub site.  I'm assuming you're planning to modify those files in GitHub to have a "Please go here" type of link so GitHub-centric folks can find them?   I'm not against the idea, but I'm still gnawing on it.

I'd a few other small nit questions on a few pages.  Also if you wanted to kill any "new" blog that's older than 3 months (just the ones with "new" in their name), I'd be fine with that.

**Comment by jberkus** [comment]
> @jberkus thanks for the heavy lifting on this one. I'd a structural change that's not clear to my noggin. I see all the *.html files got recreated under the new _site directory. Will there be any adjustments necessary for the _posts directory? 

Ooops, no, that's called "failing to completely clean up the artifacts from testing the new site".

Let me roll that back and resubmit, thanks.

Same answer for everything having to do with _site.

> I'd like to hear from @baude and @mheon about moving the install guide and tutorial here from the libpod GitHub site. I'm assuming you're planning to modify those files in GitHub to have a "Please go here" type of link so GitHub-centric folks can find them? I'm not against the idea, but I'm still gnawing on it.

Yeah, we'd want to add "tombstones" as they're called in the web biz.

> 
> I'd a few other small nit questions on a few pages. Also if you wanted to kill any "new" blog that's older than 3 months (just the ones with "new" in their name), I'd be fine with that.

That shouldn't be too hard, but should be a separate PR.

**Comment by jberkus** [comment]
DO NOT MERGE THIS.  This PR has some testing artifacts in it.  Update to come.

File: _layouts/default.html (hunk 1 above)
Comment (jberkus) [reply _layouts/default.html:27]: Blog is actually a collective noun, referring to the entire feed.  If you want to refer to the entries, those are "blog posts" or "blog entries".  The term "blogs" is grammatically incorrect.  https://www.merriam-webster.com/dictionary/blog

However, you are right about needing to eventually fix the URL path as well; I didn't want to tackle that in this PR.

Comment (TomSweeneyRedHat) [reply _layouts/default.html:27]: Well interesting, I've been using blogs incorrectly for way too long.  Thanks for the lesson and pointers!

**Comment by TomSweeneyRedHat** [comment]
@jberkus I talked to @baude today, I think his take on the move was he'd rather not move the install/tutorials from libpod but would be OK if they did move.  I need to catch up with @mheon.  However, I think both will feel better about it as long as there are "tombstones" (yet another new learning for me, ty!) left in github.com/containers/libpod.  I've long wanted the ability to have a file in a central location in GitHub yet be able to update it from more than one repo.  I know, all kinds of fun to make that happen.

Feel free to kill the "new" blogs at your leisure.  I was actually planning to do so in the near future but will hold off until the dust settles from your efforts unless it makes it easier for you.

**Comment by jberkus** [comment]
Closing this and replacing it with the clean PR #129

**Pull request closed without merging**